app/packages/components/src/components/Toast/index.ts

```diff
@@ -0,0 +1 @@
+export {default as Toast} from "./Toast";
```

Comment:
```suggestion
export { default } from "./Toast";
```
See other comment about consistency